src/test/regress/expected/multi_insert_select_conflict.out

```diff
@@ -207,18 +205,19 @@ SELECT * FROM target_table ORDER BY 1;
     10 |     0
 (10 rows)
 
-SET citus.enable_cte_inlining TO true;
 -- Test with multiple CTEs
-WITH cte AS(
+WITH cte AS MATERIALIZED (
```

Comment:
Hmm, I think in this test we specifically have `SET citus.enable_cte_inlining TO true;` so we should not make the CTEs `MATERIALIZED`?